eclipse-scout-chart/src/chart/ChartEventMap.ts

```diff
@@ -10,15 +10,27 @@
 import {Event as ScoutEvent, PropertyChangeEvent, WidgetEventMap} from '@eclipse-scout/core';
 import {AbstractChartRenderer, Chart} from '../index';
 import {ChartConfig, ChartData, ClickObject} from './Chart';
+import {LegendItem} from 'chart.js';
 
 export interface ChartValueClickEvent<C = Chart> extends ScoutEvent<C> {
   data: ClickObject;
   originalEvent?: Event;
 }
 
+export interface ChartNonValueClickEvent<C = Chart> extends ScoutEvent<C> {
+  originalEvent?: Event;
+}
+
+export interface ChartLegendItemClickEvent<C = Chart> extends ScoutEvent<C> {
+  legendItem?: LegendItem;
```

Comment:
We do not want to expose chart.js-types. Maybe its enough to simply add the legendItem-index.